core/src/avm2/globals/flash/display/stage.rs

```diff
@@ -522,3 +522,23 @@ pub fn set_full_screen_source_rect<'gc>(
     avm2_stub_setter!(activation, "flash.display.Stage", "fullScreenSourceRect");
     Ok(Value::Undefined)
 }
+
+/// Stage.fullScreenHeight's setter
+pub fn get_full_screen_height<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    avm2_stub_getter!(activation, "flash.display.Stage", "fullScreenHeight");
+    Ok(0.into())
```

Comment:
I suggest adding reasonable values here (1024x768 or 1920x1080), as this may control scaling on some games.